.vib/suitecrm/cypress/cypress/integration/suitecrm_spec.js

```diff
@@ -0,0 +1,79 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows to log in/log out', () => {
+  cy.login();
+  cy.contains('You must specify a valid username and password').should(
```

Comment:
This is nice, but as we talked, it might be interesting to check for the actual `id` of the container that shows the message, just in case it shows some other error message different than the one here.

Reply:
Yes, now the test is looking for the element which shows any type of error.